From c54421e231f2a93ed545c6d56eb5e72769b979e4 Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Tue, 25 Oct 2022 16:13:02 +0100
Subject: [PATCH] build_pip_package_with_cmake: Make compatible with yocto

We can adjust the content of build_pip_package_with_cmake.sh to allow
building using a Yocto environment.

Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh b/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
index b13a70b..b9e0421 100755
--- a/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
+++ b/tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh
@@ -88,7 +88,6 @@ case "${TENSORFLOW_TARGET}" in
       "${TENSORFLOW_LITE_DIR}"
     ;;
   aarch64)
-    eval $(${TENSORFLOW_LITE_DIR}/tools/cmake/download_toolchains.sh "${TENSORFLOW_TARGET}")
     ARMCC_FLAGS="${ARMCC_FLAGS} -I${PYBIND11_INCLUDE}"
     cmake \
       -DCMAKE_C_COMPILER=${ARMCC_PREFIX}gcc \
@@ -97,6 +96,7 @@ case "${TENSORFLOW_TARGET}" in
       -DCMAKE_CXX_FLAGS="${ARMCC_FLAGS}" \
       -DCMAKE_SYSTEM_NAME=Linux \
       -DCMAKE_SYSTEM_PROCESSOR=aarch64 \
+      -DCMAKE_SYSROOT=${POKY_SYSROOT} \
       "${TENSORFLOW_LITE_DIR}"
     ;;
   native)
-- 
2.7.4

